docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+```
+
+To update your schedule, run the previous command with a new input for schedule. To update your JSON configuration, we recommend making changes to the file, running a dry-run, and running the update command. 
+
+You can verify the updated workflow configuration by running the following show command or by clicking into your registry portal view. 
+```sh
+az acr supply-chain workflow show -r myregistry -g myresourcegroup -t continuouspatchv1
+```
+
+**NOTE:**
+Run Immediately: When Create and Update commands are executed, new Tasks will be queued based on the specified "schedule". This is behavior is intentional to avoid unnecessary patch runs for small workflow configuration updates that could incur expensive costs. If an immediate run is desired, the [--run-immediately] parameter may be provided.
+
+## Deleting the Continuous Patching Workflow
+
+To delete the continuous patching workflow, please run the following CLI command
+
+Command Schema
+```sh
+az acr supply-chain workflow delete -r <registry> -g <resourceGroup> -t continuouspatchv1 
+```
+Example Command 
+```sh
+az acr supply-chain workflow delete -r myregistry -g myresourcegroup –t continuouspatchv1
+```
+Help command to see all required/optional flags
+```sh
+az acr supply-chain workflow delete --help
+```
+
+Once a workflow is successfully deleted, the repository “csscpolicies/patchpolicy” will be automatically deleted. The 3 tasks that run your workflow will also be automatically deleted, along with any currently queued runs. 
+
+## Listing Running Tasks
+
+To list the most recently executed CSSC tasks, the following List command is available:
+```sh
+az acr supply-chain workflow list -r <registryname> -g <resourcegroup> [–-run-status <failed || successful || running>]
+```
+
+A successful result will return the following information:
+-	Image name and tag
+-	Workflow type
+-	Scan status
+-	Last scan date and time (if status failed, date would be left blank)
+-	Scan task ID (for further debugging)
+-	Patch Status
+-	Last patch date and time (if status failed, date would be left blank)
+-	Patched image name + tag
+-	Patch task ID (for further debugging)
+
+Example
+```sh
+ubuntu:jammy-20240111
+scan status: successful
+scan date: 2024-07-02T14:02:00
+scan task ID: abc
+patch status: successful
+patch date: 2024-07-02T14:04:00
+patch task id: def
+patched image: ubuntu:jammy-20240111-1
+workflow type: continuouspatchv1
+```
+The [--run-status] will return all tasks statuses that match the specified filter. This CLI command provides important debugging information.
+For example, If the "failed" value is specified under run-status, only images which have failed their patching will be listed.
+In the examples below, you’ll see a “skipped” patch status in certain scenarios. This is used to describe when a scan occurred, but no patch was required because no OS vulnerabilities were found (“Skipped” statuses are considered as successful).
+
+**Possible CLI Output Scenarios**
```

Comment:
Please also include that `EOSL` images as a possible scenario. Customers will encounter that.